Frameworks/CoreText/CTFrame.mm

```diff
@@ -83,28 +83,23 @@ void CTFrameGetLineOrigins(CTFrameRef frame, CFRange range, CGPoint origins[]) {
         range.length = 0x7FFFFFF;
     }
 
-    unsigned count = [((_CTFrame*)frame)->_lines count];
-    int idx = 0;
-
-    for (unsigned i = range.location; i < count && i < range.location + range.length; i++) {
-        origins[idx] = ((_CTFrame*)frame)->_lineOrigins[i];
-        idx++;
+    for (uint32_t index = 0; index < [((_CTFrame*)frame)->_lines count]; index++) {
```

Comment:
I think this needed a TODO that I missed. I will revert this function back to the original implementation for now

---
In reply to: [77378972](https://github.com/Microsoft/WinObjC/pull/950#discussion_r77378972) [](ancestors = 77378972)